crates/viewer/re_selection_panel/src/view_entity_picker.rs

```diff
@@ -246,3 +296,136 @@ fn add_entities_line_ui(
         }
     });
 }
+
+// ---
+
+#[derive(Debug)]
+pub struct EntityData {
```

Comment:
Bit wary of this generic name. Can we come up with something more situational? `EntityPickerEntry` or something?